meta.json

```
@@ -16,7 +16,13 @@
           "model": "viam:lorawan:node",
           "markdown_link": "README.md#configure-the-viamsensornode",
           "short_description": "A sensor compoenent for a LoRaWAN node."
-       }
+       },
+       {
+        "api": "rdk:component:sensor",
+        "model": "viam:lorawan:dragino-LHT65N",
+        "markdown_link": "README.md#configure-the-viamsensornode",
+        "short_description": "A sensor compoenent for a dragino LHT65N LoRaWAN node."
+     }
```

```suggestion
        "short_description": "A sensor component for a dragino LHT65N LoRaWAN node."
     }
```
And the short_description above too.